vendor/bat-native-ledger/src/bat_get_media.h

```diff
@@ -206,13 +205,17 @@ class BatGetMedia {
 
   std::string getYoutubeMediaKeyFromUrl(const std::string& provider_type, const std::string& media_id);
 
-  std::string getYoutubePublisherKeyFromUrl(const ledger::VisitData& visit_data);
+  static std::string getYoutubeMediaIdFromUrl(
```

Comment:
Can we move `static`s to bottom before global vars?

Reply:
moved